queries.sql

```diff
@@ -92,3 +92,56 @@ SELECT species, AVG(escape_attempts) AS average_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY species;
+
+/* query using JOIN */
+
+/* Join owner_full_name using the owners_id */
+
+SELECT animals.name
```

Comment:
- [ ] You should select all the data here not only the name, kindly make sure to fix this